tests/vmi_ignition_test.go

```diff
@@ -35,31 +35,31 @@ import (
 	cd "kubevirt.io/kubevirt/tests/containerdisk"
 )
 
-var _ = Describe("[Serial][rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:component][sig-compute]IgnitionData", func() {
+var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:component][sig-compute]IgnitionData", func() {
 
 	var err error
 	var virtClient kubecli.KubevirtClient
 
 	var LaunchVMI func(*v1.VirtualMachineInstance)
 
-	tests.BeforeAll(func() {
+	BeforeEach(func() {
 		virtClient, err = kubecli.GetKubevirtClient()
 		util.PanicOnError(err)
+		tests.BeforeTestCleanup()
+	})
 
-		LaunchVMI = func(vmi *v1.VirtualMachineInstance) {
-			By("Starting a VirtualMachineInstance")
-			obj, err := virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(util.NamespaceTestDefault).Body(vmi).Do(context.Background()).Get()
-			Expect(err).To(BeNil())
+	LaunchVMI = func(vmi *v1.VirtualMachineInstance) {
+		By("Starting a VirtualMachineInstance")
+		obj, err := virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(util.NamespaceTestDefault).Body(vmi).Do(context.Background()).Get()
+		Expect(err).To(BeNil())
 
-			By("Waiting the VirtualMachineInstance start")
-			_, ok := obj.(*v1.VirtualMachineInstance)
-			Expect(ok).To(BeTrue(), "Object is not of type *v1.VirtualMachineInstance")
-			Expect(tests.WaitForSuccessfulVMIStart(obj)).ToNot(BeEmpty())
-		}
-	})
+		By("Waiting the VirtualMachineInstance start")
+		_, ok := obj.(*v1.VirtualMachineInstance)
+		Expect(ok).To(BeTrue(), "Object is not of type *v1.VirtualMachineInstance")
+		Expect(tests.WaitForSuccessfulVMIStart(obj)).ToNot(BeEmpty())
+	}
 
 	BeforeEach(func() {
```

Comment:
Can the two beforeEach (line 45 and line 62) be merged

Reply:
Done.